src/platform/common/utils/localize.ts

```diff
@@ -214,25 +238,28 @@ export namespace DataScience {
 
     export const dataExplorerTitle = localize('DataScience.dataExplorerTitle', 'Data Viewer');
     export const badWebPanelFormatString = localize(
-        'DataScience.badWebPanelFormatString',
+        {
+            key: 'DataScience.badWebPanelFormatString',
+            comment: ['{RegEx=">[^<]*<"}', 'Only translate the text within the HTML tags']
+        },
         '<html><body><h1>{0} is not a valid file name</h1></body></html>'
     );
     export const checkingIfImportIsSupported = localize(
-        'DataScience.checkingIfImportIsSupported',
-        'Checking if import is supported'
+        { key: 'DataScience.checkingIfImportIsSupported', comment: ['{Locked="import"}'] },
+        'Checking if "import" is supported'
     );
     export const installingMissingDependencies = localize(
         'DataScience.installingMissingDependencies',
         'Installing missing dependencies'
     );
     export const validatingKernelDependencies = localize(
-        'DataScience.validatingKernelDependencies',
+        { key: 'DataScience.validatingKernelDependencies', comment: ['{Locked="kernel"}'] },
         'Validating kernel dependencies'
     );
     export const performingExport = localize('DataScience.performingExport', 'Performing Export');
     export const convertingToPDF = localize('DataScience.convertingToPDF', 'Converting to PDF');
     export const exportNotebookToPython = localize(
-        'DataScience.exportNotebookToPython',
+        { key: 'DataScience.exportNotebookToPython', comment: ['{Locked="Notebook"}'] },
```

Comment:
I am assuming we want to keep uppercase `Notebook` as is and not change it from language to language.

However, I am also assuming that sometimes it's ok to translate `notebook`.

Let me know if I should keep all of the `notebook` instances as `Locked`.